### PR TITLE
My awesome commit

### PR DESCRIPTION
**Pull request by CalinBolea**

### CODE REVIEW

**Comment by marceldobrica** [comment]
Awesome....

**Comment by comanpatricia** [comment]
really awesome!

**Comment by denis-crasnean** [comment]
test

**Review by rusdavid1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# training-2022
\ No newline at end of file
+# training-2022
+
+
+asfsrfrsfrs
```
Comment (CalinBolea) [inline README.md]: bai, linia asta nu-i buna

Comment (OvidiuGi) [inline README.md]: aici nu e bine

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
-# training-2022
\ No newline at end of file
+# training-2022
+
+
```
Comment (rusdavid1) [inline README.md:3]: dsafwefwf

File: README.md (hunk 1 above)
Comment (denis-crasnean) [inline README.md]: How many WTF per minute?

File: README.md [hunk 3]
```diff
@@ -1 +1,4 @@
...
+acum e cald afara?
```
Comment (rusdavid1) [inline README.md:5]: maybe

**Pull request merged**